src/js/control-bar/seek-to-live.js

```diff
@@ -39,19 +39,19 @@ class SeekToLive extends Button {
       className: 'vjs-seek-to-live-control vjs-control'
     });
 
-    this.contentEl_ = Dom.createEl('div', {
-      className: 'vjs-seek-to-live',
-      innerHTML: `<span class="vjs-control-text">${this.localize('Stream Type')}\u00a0</span>${this.localize('LIVE')}`
+    this.textEl_ = Dom.createEl('span', {
+      className: 'vjs-seek-to-live-text',
+      innerHTML: this.localize('LIVE')
     }, {
-      'aria-live': 'off'
+      'aria-hidden': 'true'
     });
 
     this.circleEl_ = Dom.createEl('span', {
       className: 'vjs-seek-to-live-circle'
```

Comment:
while we're at it, should we just reuse the vjs-icon-placeholder element for this? Probably fine to leave it as is, though.

Reply:
yeah let me see if that has any unknown side effects

Reply:
seems better to me, implemented 👍